### PR TITLE
chore: log r2d2 to other.log

### PR DESCRIPTION
**Pull request by Cifko**
Description
---
Append `r2d2` log to `other.log`

### CODE REVIEW

**Review by philipr-za — DISMISSED** [review]
LGTM

**Review by philipr-za — COMMENTED** [review]
Oh actually, could you please add it to the wallet file too.

**Comment by Cifko** [comment]
> Oh actually, could you please add it to the wallet file too.

Done

File: common/logging/log4rs_sample_base_node.yml [hunk 1]
```diff
@@ -146,3 +146,8 @@ loggers:
     level: info
     appenders:
       - base_layer
+  # Route R2D2 log events
+  r2d2:
+    level: warn
+    appenders:
+      - other
```
Comment (stringhandler) [inline common/logging/log4rs_sample_base_node.yml:153]:
```suggestion
      - other
   additive: false   
```

**Review by stringhandler — COMMENTED** [review]
Add "additive=false" to remove them from the other logs (e.g. console)

**Review by stringhandler — APPROVED** [review]

**Pull request merged**